src/main/java/com/kk/strong/security/CustomAuthorizationFilter.java

```diff
@@ -0,0 +1,73 @@
+package com.kk.strong.security;
+
+import com.auth0.jwt.JWT;
+import com.auth0.jwt.JWTVerifier;
+import com.auth0.jwt.algorithms.Algorithm;
+import com.auth0.jwt.interfaces.DecodedJWT;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.security.authentication.UsernamePasswordAuthenticationToken;
+import org.springframework.security.core.authority.SimpleGrantedAuthority;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.web.filter.OncePerRequestFilter;
+
+import javax.servlet.FilterChain;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Collection;
+
+import static java.util.Arrays.stream;
+
+@Slf4j
+public class CustomAuthorizationFilter extends OncePerRequestFilter {
```

Comment:
`Custom` is a very generic word, maybe include `Security` and `Token` keywords in the name of this filter?

Reply:
`Authorization` however is a very good keyword, so my problem is only to `Custom`